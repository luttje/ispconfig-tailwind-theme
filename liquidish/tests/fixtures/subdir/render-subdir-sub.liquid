<div>
    {% render '../render-basic-sub', slot: 'test' %}
</div>
