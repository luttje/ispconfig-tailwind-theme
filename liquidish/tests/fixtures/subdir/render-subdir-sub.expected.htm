<div>
    <h1 class="text-2xl">test</h1>
</div>
